README.md

```diff
@@ -21,8 +21,8 @@ Verify links in [awesome](status/status.md) projects
 ``` shell
 Usage: awesome_bot [file or files]
        awesome_bot [options]
-    -f, --files [files]              Comma separated files to check
-    -a, --allow [errors]             Status code errors to allow
+    -f, --files [files]              Comma-separated files to check
+    -a, --allow [errors]             Status-code errors to allow
```

Comment:
Usage comes from the script so please also make the change in the script or undo, thanks